### ISSUE TITLE
"try again in an hour" doesn't work at all

### ISSUE THREAD

**Issue by MClatter** [remnoteio/remnote-issues#273]
**Describe the bug**
When I click on "accidentally pressed 'Show answer' too early try again in an hour" button the flashcard disappears from the queue indefinitely.

**Desktop:**
 - OS: Windows 10
 - Browser: Firefox
 - Version: 82.0.3 (64-bit)

**Additional context**
It would be a very useful feature for me if it worked. I noticed when I see a hard card to recall in Anki, I just hit 's' shortcut and the card gets shuffled for me to see it later randomly. The more I see it and shuffle it the better chances I have to finally recall it later.
I wish I could use Remnote the same way.

**Comment by tfiwm** [comment]
Are you able to consistently reproduce this issue? Is this actually due to this underlying issue #262 ?

**Comment by MClatter** [comment]
> 
> 
> Are you able to consistently reproduce this issue? Is this actually due to this underlying issue #262 ?

Yes. I chose two recent Rems to test the issue. Both of them disappeared. I believe several days past since. Couple days ago I
searched through my Rems for forgotten ones and switched them from "Forward only" to "None" and back to "Forward only". Some of them I also edited. Well, several of them appeared in the queue immediately but not the two recent ones that I had specifically tested. Those two I didn't edit though, only switched them "Forward only"/"None"/"Forward only" and made them Concepts instead of Descriptors. I don't follow the rule of capitalizing concepts but I doubt it's problematic.

Unlike #262 I don't use Remnote on Android yet. I did not modify the algorithm as well. I would gladly send my Rems, stats etc. if requested.
